target/scripts/startup/setup.d/dovecot.sh

```diff
@@ -40,6 +40,20 @@ function _setup_dovecot
 
   esac
 
+  if [[ ${ENABLE_POP3} -eq 1 ]]
+  then
+    _log 'debug' 'Enabling POP3 services'
+    mv /etc/dovecot/protocols.d/pop3d.protocol.disab /etc/dovecot/protocols.d/pop3d.protocol
+  fi
+
+  [[ -f /tmp/docker-mailserver/dovecot.cf ]] && cp /tmp/docker-mailserver/dovecot.cf /etc/dovecot/local.conf
+}
+
+function _setup_dovecot_sieve
+{
+  mkdir -p /usr/lib/dovecot/sieve-{filter,global,pipe}
+  mkdir -p /usr/lib/dovecot/sieve-global/{before,after}
```

Comment:
Just a note that there is no more `rm -f` cleanup on these locations if they contain anything (_not that they should, but was part of previous restart cleanup behaviour I assume_).

Likewise for `.sieve` and `.svbin` files in `sieve-global/{before,after}`.

Reply:
That is intentional. I found no good reason for deleting all files there. And it works flawlessly without. I found that the dirs are empty to begin with, so the cleanup is irrelevant.